src/NHSD.BuyingCatalogue.Ordering.Api/Services/CompleteOrder/CompleteOrderService.cs

```diff
@@ -13,17 +15,20 @@ public sealed class CompleteOrderService : ICompleteOrderService
         private readonly IIdentityService _identityService;
         private readonly IOrderRepository _orderRepository;
         private readonly IEmailService _emailService;
+        private readonly IPurchasingDocumentService _purchasingDocumentService;
```

Comment:
The name of the service so far has matched the name of the use case e.g. folder name.